cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -0,0 +1,471 @@
+#include <errno.h>
+#include <stdio.h>
+
+#include "net/gnrc.h"
+
+#include "cc2538_rf.h"
+#include "cc2538_rf_internal.h"
+
+#include "net/ieee802154/radio.h"
+
+#define CC2538_ACCEPT_FT_2_ACK     (1 << 5)
+
+#if IS_USED(MODULE_IEEE802154_RADIO_HAL)
+static const ieee802154_radio_ops_t cc2538_rf_ops;
+
+ieee802154_dev_t cc2538_rf_dev = {
+    .driver = &cc2538_rf_ops,
+};
+
+static uint8_t cc2538_min_be = 3;
+static uint8_t cc2538_max_be = 5;
+static int cc2538_retries = 4;
+
+static bool cc2538_cca_status;
+static bool cc2538_cca;
```

Comment:
yes, I will add a comment.

Reply:
ah, it actually means that the result of the CCA is due to a CCA request and not because of a failed CSMA-CA (just to distinguish which kind of confirmation should be generated). I already documented this.

Reply:
done!